fake-toolchain/Unix/cc

```diff
@@ -9,6 +9,6 @@ elif [ "$1" = "-v" ] || [ "$1" = "--version" ]; then
     $YCM_CONFIG_GEN_CC_PASSTHROUGH $@
 
 else
-    echo "$@" >> $YCM_CONFIG_GEN_CC_LOG
+	echo "YCM_CONFIG_GEN_PWD=$(pwd) $@" >> $YCM_CONFIG_GEN_CC_LOG
```

Comment:
You need to quote the directory here, otherwise this will fail if it contains spaces.